src/components/BestScores.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+type Best = {
+  language: string;
+  accuracy: number;
+  speed: number;
+};
+
+type Props = {
+  data: Best[];
+};
+
+const BestScores = (props: Props) => {
+  const { data } = props;
```

Comment:
```suggestion
const BestScores: React.VFC<Props> = ({data}) => {
```
moriyasuさんの書き方でも大丈夫ですが、今回は上の書き方で統一してもらっていいですか？
- React.VFC<Props> -> reactの関数コンポーネントで、propsの型はPropsだよと明示している
- {data} -> propsから分割代入をして、直接引数に渡している

Reply:
props への型の渡し方を上記の方法に統一する件、承知しました。